src/renderer/player.ts

```diff
@@ -132,7 +132,7 @@ export default class Player {
 		this.state.richPresenceTimer && clearInterval(this.state.richPresenceTimer);
 		this.state.richPresenceTimer = setInterval(() => {
 			this.state.currentlyPlayingMetadata && this.electron.invoke("update-rich-presence", [
-				this.state.currentlyPlayingMetadata.common.artist ? `${this.state.currentlyPlayingMetadata.common.artist || "Unkown Artist"} - ${this.state.currentlyPlayingMetadata.common.title}` : this.state.currentlyPlayingFilePath.substring(this.state.currentlyPlayingFilePath.lastIndexOf("\\") + 1),
+				this.state.currentlyPlayingMetadata.common.albumartist ? `${this.state.currentlyPlayingMetadata.common.albumartist!} - ${this.state.currentlyPlayingMetadata.common.title}` : this.state.currentlyPlayingFilePath.substring(this.state.currentlyPlayingFilePath.lastIndexOf("\\") + 1),
```

Comment:
this seems to have gotten somehow reverted, i had done this commit after you forked it for this PR https://github.com/amethyst-px/amethyst/commit/ed0c42ce4efd19b624def304c92acd6cad30e82c

Reply:
Ye I had some weird issues when I pulled from the fork, don't exactly know why but that might be the cause